src/test/utils/nc_test_utils.js

```diff
@@ -1,7 +1,7 @@
 /* Copyright (C) 2016 NooBaa */
 'use strict';
 
-const test_utils = require('./test_utils');
+const test_utils = require('../system_tests/test_utils');
```

Comment:
please move to nc, this is a util for the nc tests and not a test file that tests utils